### DISCUSSION TITLE
About Font Autumn

### DISCUSSION THREAD

**Discussion by arvislacis** [wintercms/winter#228] [General]
In some places I can see references to "Font Autumn":

1. https://github.com/wintercms/wn-builder-plugin/blob/main/lang/en/lang.php#L648
2. https://github.com/wintercms/winter/blob/develop/modules/system/assets/ui/less/icon.mixins.less#L9
3. In some other .less files
 
 As I understand then it's mostly only a wrapper for Font Awesome 4.7 icons...
 So the question is - **Is it really needed?**
 
I mean - I understand that it's wrapper and it's ok to use and reference to it in .less files (2./3. points in the list) but, for example, in the Builder plugin (1. point in the list) the sentence `Winter CMS uses Font Autumn icons` may only cause additional misunderstanding for users as it should probably be replaced with something like `Winter CMS uses wrapper for Font Awesome icons`.

Anyways, because of Winter CMS fork this should probably be called `Font Winter`. :smiley:

**Comment by LukeTowers** [comment]
It would be nice to revamp the whole icon suite in the first place (either by updating to a newer version of Font Awesome or evaluating other icon libraries) but at the moment it's just not a high priority for us. Feel free to submit a PR to update the language file though!
